db/hints/manager.cc

```diff
@@ -40,7 +40,6 @@
 #include "database.hh"
 #include "service_permit.hh"
```

Comment:
If you revert an entire merge, we can revert the merge commit. But we still need to explain it.

Reply:
Not everything is reverted. Unless I missed something, those are parts which are kept:

- RPC verbs are kept because they are used in this PR,
- The machinery which allows the hints manager to wait until hints are replayed - although it is modified a bit by commit https://github.com/scylladb/scylla/pull/8728/commits/2059077d5cf94765fa767e8b1e0520841b4d2bdb,
- Some small parts in repair code is now used by new code unrelated to this feature (e.g. get_hosts_participating_in_repair function, abort sources).

Reply:
Ok.